### ISSUE TITLE
Neighbors incorrectly detected with bgplgd source type

### ISSUE THREAD

**Issue by jof** [alice-lg/alice-lg#135]
I just upgraded from 6.0.0 to the fresh 6.1.0 release and discovered that there appears to be a regression for bgplgd-type backend sources.

When I list neighbors for the bgplgd sources, it shows neighbor IPs, names, and route counts correctly (from the `/neighbors`) method. However, selecting a neighbor for more detail appears to template the neighbor name/IP as an empty string. Relatedly, bgplgd is receiving an API request for RIB data like `/rib?neighbor=` (empty neighbor parameter), and errors appear in the alice log like:
```
Refeshing neighbors of xxxxxxxx failed: unexpected end of JSON input
```

These errors are no longer present after rolling back to version 6.0.0

**Comment by annikahannig** [comment]
Bugfix release pending. :)

Thank you for finding this.

**Comment by jof** [comment]
This particular bug is fixed with the merge of #137, though a versioned release tag incorporating the fix would not be unwelcome.
(however, we are un-broken for now with a private fork-branch)
